bootloader/mockbootloader/mockbootloader.go

```diff
@@ -17,7 +17,7 @@
  *
  */
 
-package boottest
+package mockbootloader
```

Comment:
this should be bootloadertest for consistency

Reply:
done